probers/ldap/probe.go

```diff
@@ -0,0 +1,33 @@
+package ldap
+
+import (
+	"github.com/nmcclain/ldap"
```

Comment:
Why this repository rather than one of the many other Go LDAP repositories on GitHub? I also note that this one is a an N-fork of mmitton/ldap. Why not at least use the upstream one?